app/packages/core/src/components/Actions/Similar.tsx

```diff
@@ -237,91 +291,201 @@ const SortBySimilarity = React.memo(
       current && setState(current);
     }, [current]);
 
+    useEffect(() => {
+      if (!hasNoSelectedSamples) {
+        setIsImageSearch(true);
+      }
+    }, [hasNoSelectedSamples]);
+
+    const popoutStyle = { minWidth: 280 };
+
+    const onInfoLink = () => {
+      window.open(SORT_BY_SIMILARITY, "_blank");
```

Comment:
This is not allowed. You'll have to use `useExternalLink` to ensure it will work in the desktop app.